src/content/vintage/getting-started/observability/logging/architecture/index.md

```diff
@@ -35,10 +35,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scrapping agents run on the management cluster and your workload clusters alike. There is different tools for different purposes:
+    - Promtail is used in older versions to retrieve the container and kubernetes audit logs.
+    - Alloy is used in newer versions to retrieve the container and kubernetes audit logs.
     - Grafana Agent is used to retrieve the kubernetes events.
 
```

Comment:
But according to the picture, Alloy replaces Promtail and Grafana in CAPI v29? or only promtail.

Reply:
It only replaces Promtail for now, but will eventually replace Grafana agent in the future.